fetch_binary_drivers/package.xml

```diff
@@ -40,7 +40,7 @@
   <!-- system dependencies which the drivers dyanmically link against -->
   <depend>boost</depend>
   <depend>curl</depend>
-  <depend>python</depend>
+  <depend>python3</depend>
```

Comment:
If you want the same branch to support both noetic (and earlier) you can upgrade to package format 3 and use conditionals (see https://github.com/fetchrobotics/robot_controllers/blob/melodic-devel/robot_controllers/package.xml#L29)

Reply:
I had been planning on keeping melodic_devel for any melodic releases.  I'm wary of complexity (which in this case is anything new beyond the minimum needed 😬 ) given my limited time to test much.